std/typecons.d

```diff
@@ -1168,12 +1168,12 @@ if (distinctFieldNames!(Specs))
             Returns:
                 A `size_t` representing the hash of this `Tuple`.
          */
-        size_t toHash() const nothrow @safe
+        size_t toHash() const nothrow @trusted
```

Comment:
This can't be legitimately marked `@trusted` because `hashOf(x)` can call `x.toHash()` which may do any number of unsafe things. It also can't be `@trusted` because of the "Object.toHash() is not const" fiasco.